### PR TITLE
DEV-6358 agency v2 FY Summary section scaffolding

### PR DESCRIPTION
**Pull request by ebdabbs**
**High level description:**

Implements the layout of the Agency v2 FY Summary section.

**Technical details:**

- Using mock data to avoid duplicate data fetching while the visualizations are also in progress

**JIRA Ticket:**
[DEV-6358](https://federal-spending-transparency.atlassian.net/browse/DEV-6358)

**Mockup:**
Mobile: https://bahdigital.invisionapp.com/share/Z9IAG5XBJQY
Desktop: https://bahdigital.invisionapp.com/share/WUIAG3CQ4JG#/296056413_agency-Profile-Mvp

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- N/A Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` - LMK if you disagree!
- [x] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-transparency-ui/pull/89) updated

Reviewer(s):
- [x] Design review complete @marcyheld 
- [x] Code review complete

### CODE REVIEW

File: src/_scss/pages/agencyV2/overview/_visualizationSection.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.visualization-section {
+    text-align: center;
+    height: 100%;
+    @supports not (-ms-high-contrast: none) {
```
Comment (ebdabbs) [inline src/_scss/pages/agencyV2/overview/_visualizationSection.scss:4]: IE does not correctly handle Flexbox `flex-direction: column`, hence the `@supports` query (which is also incompatible with IE) to selectively apply those styles to bottom-align the visualization labels. This is kind of hacky but it works to get the alignment we want in other browsers while keeping IE from looking terrible.

**Review by brett-varney — CHANGES_REQUESTED** [review]
Overall I approve; I see a couple of mobile-view items that might be rejected. The carousel frames should have a header that I don't see a place for ("FY 2019 Budget", "FY 2019 Obligations" etc in mocks), and I don't see the text line breaks in mobile subtitles, like isMobile is not being recognized by VisualizationSection. But I think everything else works.

**Comment by ebdabbs** [comment]
Good observation; I talked to design and those headers are not part of Agency v2. We might use them in other profile pages in the future but I did confirm we should leave them out. Marcy requested that the line breaks be in desktop (so they are all two lines at standard desktop width) and not to break up the text in mobile when there may be more horizontal space available since only one is visible at a time.

**Comment by brett-varney** [comment]
> 
> 
> Good observation; I talked to design and those headers are not part of Agency v2. We might use them in other profile pages in the future but I did confirm we should leave them out. Marcy requested that the line breaks be in desktop (so they are all two lines at standard desktop width) and not to break up the text in mobile when there may be more horizontal space available since only one is visible at a time.

Should AC be updated, or does QA know that already?

**Comment by ebdabbs** [comment]
@brett-varney Added a testing note to the ticket!

**Review by brett-varney — APPROVED** [review]

**Pull request merged**